src/app/Plans/components/PlansTable.tsx

```diff
@@ -66,10 +88,29 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
 
   currentPageItems.forEach((plan: IPlan) => {
     let buttonText: string | React.ReactNode;
-    if (plan.status.conditions.every((condition) => condition.type === 'Ready')) {
+    let isStatusReady = false;
+    let title = '';
+    let variant: ProgressVariant | undefined;
+
+    const hasCondition = (type) => {
```

Comment:
I am ok committing this for now. Lets open an issue to create a fine tuned this status helper function & dig deeper into how status will work with the MTV API.

Reply:
Agreed. Let's put a `: string` type annotation on that `type` param though.